docs/docs/cmd/spo/group/group-member-remove.md

```diff
@@ -22,18 +22,36 @@ m365 spo group member remove [options]
 `--userName <userName>`
 : The UPN of the user that needs to be removed (user principal name, eg. megan.bowen@contoso.com)
 
+`--aadGroupId [aadGroupId]`
+: The object Id of the Azure AD group to remove as a member. Specify either `aadGroupId` or `aadGroupName`.
+
+`--aadGroupName [aadGroupName]`
+: The name of the Azure AD group to remove as a member. Specify either `aadGroupId` or `aadGroupName`.
```

Comment:
I think we should update the description that we may remove either by `userName` or `aadGroupId` or `aadGroupName`